packages/web-components/src/mixins/headers.css

```diff
@@ -11,3 +11,11 @@
 :host label :is(h1, h2, h3, h4, h5)  {
   font-family: var(--font-serif);
 }
+
+:host h1 + *, :host h2 + *, :host h3 + *, :host h4 + *, :host h5 + *, :host h6 + * {
+  margin-top: unset;
+}
+
+:host #form-question {
+  margin-bottom: 1rem;
```

Comment:
This keeps a bottom margin on the header when it's used with the forms pattern in this way: 

 
<img width="382" alt="Screenshot 2024-06-28 at 1 34 32 PM" src="https://github.com/department-of-veterans-affairs/component-library/assets/872479/c278e609-6091-4d43-b828-093fdb5b600f">